app/src/main/res/layout/activity_marketing.xml

```diff
@@ -35,6 +35,7 @@
         android:layout_marginEnd="@dimen/base_margin_double"
         android:layout_marginBottom="@dimen/base_margin"
         android:text="@string/MARKETING_SCREEN_CTA"
+        android:textColor="?textColorButton"
```

Comment:
This should not be necessary, this color should be applied automatically through `style="?materialButtonContainedLargeStyle"`

Reply:
yea, i thought so too but i does not seem to work, but i get a white text when on light theme even if we have `<item name="textColorButton">@color/hedvig_black</item>`

Reply:
This is actually intended, check out the `ThemeOverlay` applied to the root of this Activity. Basically, this screen only has white text no matter what in accordance with a decision from design.